### DISCUSSION TITLE
disabling the icon

### DISCUSSION THREAD

**Discussion by Abel808** [t184256/gnome-shell-extension-autohide-volume#3] [Q&A]
Hi, how could i make the icon disabled regardless of its volume state. i couldn't figure it out as i don't know much coding.

**Comment by t184256** [comment]
https://askubuntu.com/questions/1226492/how-can-i-hide-volume-control-in-gnome-shell suggests you don't need to write extensions for it, just autostart a command that hides it.

Also sounds like a good feature request for Just Perfection extension.

**Comment by t184256** [comment]
Ah, no, Gnome doesn't allow calling Eval anymore.

Anyway, sounds like `Main.panel.statusArea.quickSettings._volume.hide()` is the only thing your extension would be doing.
